modules/apps/adaptive-media/adaptive-media-blogs-web/src/main/java/com/liferay/adaptive/media/blogs/web/internal/counter/BlogsAMImageCounter.java

```diff
@@ -28,26 +20,8 @@
 public class BlogsAMImageCounter extends BaseAMImageCounter {
 
 	@Override
-	protected void forEachFileEntry(
-			long companyId, Consumer<DLFileEntry> consumer)
-		throws PortalException {
-
-		_dlFileEntryLocalService.forEachFileEntry(
-			companyId,
-			_classNameLocalService.getClassNameId(BlogsEntry.class.getName()),
-			consumer,
-			_dlFileEntryConfigurationProvider.
-				getCompanyPreviewableProcessorMaxSize(companyId),
-			getMimeTypes());
+	protected long getClassNameId() {
+		return classNameLocalService.getClassNameId(BlogsEntry.class.getName());
 	}
```

Comment:
Instead of this, can you make subclasses return the class name as a string (or null if none), and let the base class invoke `classNameLocalService`? That way, subclasses don't have to both override and access the base classes field. E.g.:
```
@Override
protected String getClassName() {
    return BlogsEntry.class.getName();
}
```